app/views/additional_payments/claims/supply_teacher.html.erb

```diff
@@ -1,45 +1,38 @@
-<% content_for(:page_title, page_title(t("additional_payments.questions.employed_as_supply_teacher"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
+<% content_for(:page_title, page_title(t("#{@form.i18n_namespace}.forms.supply_teacher.questions.employed_as_supply_teacher"), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
 <% path_for_form = current_claim.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.employed_as_supply_teacher": "claim_eligibility_attributes_employed_as_supply_teacher_true" }) if current_claim.errors.any? %>
-    <%= form_for current_claim, url: path_for_form do |form| %>
-      <%= form_group_tag current_claim do %>
-        <%= form.fields_for :eligibility, include_id: false do |fields| %>
+    <%= render("shared/error_summary", instance: @form, errored_field_id_overrides: { "eligibility.employed_as_supply_teacher": "claim_eligibility_attributes_employed_as_supply_teacher_true" }) if @form.errors.any? %>
 
-          <%= fields.hidden_field :employed_as_supply_teacher %>
+    <%= form_for @form, url: path_for_form, method: :patch do |form| %>
+      <%= form_group_tag @form do %>
+        <%= hidden_field_tag :employed_as_supply_teacher, params[:employed_as_supply_teacher] %>
```

Comment:
Does
```suggestion
        <%= form.hidden_field :employed_as_supply_teacher %>
```
work? If so might be worth considering to simplify the view a little

Reply:
oo yes! it does, thanks